Tests/test_geo.py

```diff
@@ -5,20 +5,15 @@
 
 """Tests the basic functionality of the GEO parsers."""
 
-import sys
-
 import unittest
 
 from Bio import Geo
 
+import builtins
 
-if sys.version_info[0] >= 3:
-    # Python 3 problem: Can't use utf8 on Tests/Geo/soft_ex_*.txt
-    # due to micro (\xb5) and degrees (\xb0) symbols
-    import builtins
 
-    def open(path):
-        return builtins.open(path, encoding="latin")
+def open(path):
+    return builtins.open(path, encoding="latin")
```

Comment:
For clarity I would prefer to just add `, encoding="latin"` to all calls to `open`.

Reply:
sorry, I am not following you. can you please explain your concern a bit?

Reply:
For example, on line 28(old)/23(new) you could use
```
         with open(path, encoding="latin") as handle:
```
Then you don't need to define the `open` function on line 15-16.

Reply:
I agree, that would be nicer than redefining the builtin function ``open`` which was done with the constraints of supporting Python 2 and 3 at the same time.

Reply:
Fixed this